pallets/anchors/src/common.rs

```diff
@@ -21,14 +21,19 @@ use frame_support::storage::child::ChildInfo;
 pub const MILLISECS_PER_DAY: u64 = 86400000;
 
 /// Get days(round up) since epoch given the timestamp in ms
-pub fn get_days_since_epoch(ts: u64) -> u32 {
+pub fn get_days_since_epoch(ts: u64) -> Option<u32> {
 	let remainder = ts % MILLISECS_PER_DAY;
-	let days = (ts / MILLISECS_PER_DAY) as u32;
-	if remainder == 0 {
-		days
-	} else {
-		days + 1
-	}
+	let days_option = u32::try_from(ts / MILLISECS_PER_DAY).ok();
+	return match days_option {
+		None => None,
+		Some(v) => {
+			if remainder == 0 {
+				Some(v)
+			} else {
+				v.checked_add(1)
+			}
+		}
+	};
```

Comment:
`til` - I think we can simplify this to:

```suggestion
	let days = u32::try_from(ts / MILLISECS_PER_DAY).ok()?;
  
  if remainder == 0 {
	  Some(days)
	 } else {
    days.checked_add(1)
   }
```

Reply:
Nice, thanks I didnt know that `?` returned an option in addition to a result 👍

Reply:
yeah I think it's a somewhat new feature I believe :)